src/harness/testcases/WINNF_FT_S_MCP_testcase.py

```diff
@@ -101,55 +101,50 @@ def generate_MCP_1_default_config(self, filename):
     pal_low_frequency = 3550000000.0
     pal_high_frequency = 3560000000.0        
 
-    ppa_records_1, pal_records_1 = makePpaAndPalRecordsConsistent(ppa_record,
+    ppa_record_1, pal_records_1 = makePpaAndPalRecordsConsistent(ppa_record,
                                                                 [pal_record],
                                                                 pal_low_frequency,
                                                                 pal_high_frequency,
                                                                 'test_user_1')
-    # Load DPA record
+    # Define DPA's 
     dpa_1 = {
-       'dpaId': 'test_dpa_id1',
+       'dpaId': 'dpa_east_4',
        'frequencyRange': {'lowFrequency': 3650000000 , 'highFrequency': 3660000000 }
     }
     dpa_2 = {
-       'dpaId': 'test_dpa_id2',
+       'dpaId': 'dpa_east_5',
        'frequencyRange': {'lowFrequency': 3660000000 , 'highFrequency': 3670000000 }
     }
     dpa_3 = {
-       'dpaId': 'test_dpa_id3',
+       'dpaId': 'dpa_east_6',
        'frequencyRange': {'lowFrequency': 3670000000 , 'highFrequency': 3680000000 }
     }
 
-    # Define conditionals
-    conditionals = {
-        'registrationData': [conditionals]
-    }
-
     # Registration and grant records for multiple iterations
-    cbsd_records_iteration_1_domain_proxy_0 = {
+    cbsd_records_iteration_0_domain_proxy_0 = {
         'registrationRequests': [device_1, device_2],
         'grantRequests': [grant_request_1, grant_request_2]
     }
-    cbsd_records_iteration_1_domain_proxy_1 = {
+    cbsd_records_iteration_0_domain_proxy_1 = {
         'registrationRequests': [device_3],
         'grantRequests': [grant_request_3]
     }
-    cbsd_records_iteration_2_domain_proxy_0 = {
+    cbsd_records_iteration_1_domain_proxy_0 = {
         'registrationRequests': [device_4],
         'grantRequests': [grant_request_4]
     }
-    cbsd_records_iteration_2_domain_proxy_1 = {
+    cbsd_records_iteration_1_domain_proxy_1 = {
         'registrationRequests': [device_5, device_6],
         'grantRequests': [grant_request_5, grant_request_6]
     }
     
     # Protected entities records for multiple iterations
-    protected_entities_iteration_1 = {
+    protected_entities_iteration_0 = {
         'palRecords': pal_records_1,
-        'ppaRecords': ppa_records_1, 
+        'ppaRecords': ppa_record_1, 
```

Comment:
This needs to be a list. Please be more careful.

Reply:
updated   as lists